config/experiments/scrabble/jay.yaml

```diff
@@ -51,21 +52,21 @@ policy:
     shared_weights: False
     checkpoint: backward
 
+# Evaluator
```

Comment:
I am unsure what this comment refers to exactly, but I would just say that the icml23/ctorus.yaml file is really old (January 2023) so it would be fine to deprecate it / adapt it if needed. Yes, it contains the experiments of a paper, but I believe it's ok to adapt it to the new state of the repo.